docs/reference/changelog.md

```diff
@@ -4,13 +4,96 @@ title: Changelog
 sidebar_label: Changelog
 ---
 
+## Darklang Release 4 (August 1, 2022)
+
+- Many standard library functions now have better descriptions
+  ([#4273](https://github.com/darklang/dark/pull/4273)).
+
+- Add new version of `Result::fromOption`, which allows non-string errors
+  ([#4272](https://github.com/darklang/dark/pull/4272)).
+
+- Add new version of `Bytes::base64Decode`, which fails properly on invalid
+  input ([#4287](https://github.com/darklang/dark/pull/4287)).
+
+- Bug fix: Dates were offset in Analysis by the number of hours away the user is
+  from GMT ([#4260](https://github.com/darklang/dark/pull/4260)).
+
+### Rewrite Dark in Dark
+
+Much of ApiSever (the backend that powers the editor) is being rewritten in
+Dark.
+
+- Expose more internal staticassets fns for uploads
+  ([#4278](https://github.com/darklang/dark/pull/4278)).
+
+- Switch Presence URL to dark-editor canvas
+  ([#4294](https://github.com/darklang/dark/pull/4294)).
+
+### Work towards future improvements
+
+- Add StdLib fns for Tuples
+  ([#4311](https://github.com/darklang/dark/pull/4311)). While tuples are not
+  yet available in the Editor, we're making steady progress and hope to have
+  them available next release.
+
+- Provide more metadata to the client regarding standard library functions
+  ([#4277](https://github.com/darklang/dark/pull/4277)). In a follow up PR, the
+  editor will leverage this data to display additional contextual data around
+  functions being used.
```

Comment:
What do you think about this being a section?

Some PRs don't have immediate use to users, but also aren't "Internal" improvements. I like this because it gives a sneak preview, but open to thoughts